graph/protocol.go

```diff
@@ -38,9 +38,9 @@ type Protocol struct {
 // Each supported protocol is defined below.  Each rate provided as node or edge metadata must be defined.
 // Each method below should have a section handling each supported protocol.
 
-//
-// GRPC Protocol
-//
+/*
+GRPC Protocol
+*/
```

Comment:
same as above, I don't think we should be changing the comment style of the original author, unless there is some good reason.  Files are being updated in this PR just to change comment style or indentation.